tools/s3segmenter/s3segmenter.xml

```diff
@@ -0,0 +1,192 @@
+<tool id="s3segmenter" name="s3segmenter" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>S3segmenter is a Python-based set of functions that generates single cell (nuclei and cytoplasm) label masks.</description>
```

Comment:
too large for a description imho, no one cares if it is Python based here, this can go to the help section.

`<description>single cell (nuclei and cytoplasm) label masks</descriptions>`